.gitignore

```diff
@@ -67,3 +67,5 @@ twistd.pid
 .metadata/
 
 .DS_Store
+/octopus/manufacturer/vapourtec.py
+/octopus/blocktopus/blocks/vapourtec.py
```

Comment:
Can you keep this in your own repository please and not commit it? 
This is meant to be the purpose of plugins, though that system should probably be improved :-) (or documented...)

Reply:
> Can you keep this in your own repository please and not commit it?
> This is meant to be the purpose of plugins, though that system should probably be improved :-) (or documented...)

Sure, actually I use those as plugins but sometimes is faster to just copy and paste and I added those to gitignore as to be 100% sure that they never get committed by mistake as once they are in the git history it is hard to remove them

Reply:
Ah - that makes sense! Still, this shouldn't go into the main repository. I'd recommend putting a .gitignore file in a subdirectory (either in octopus/, or in both octopus/manufacturer and octopus/blocks) with these in and then not committing that .gitignore.